### ISSUE TITLE
Message of Electron message boxes isn't being read by JAWS 2021, but is read by JAWS 2020

### ISSUE THREAD

**Issue by pushkin-** [FreedomScientific/VFO-standards-support#518]
### Summary

Not sure if there's a simpler way to repro for you, but I have an Electron app that shows a dialog box:

main.js:
```
(async () =>
{
	const { app, dialog } = require("electron");
	await app.whenReady();
	dialog.showMessageBoxSync(null, {
		message: "test message",
		title: "test title",
		type: "info",
		buttons: ["ok", "cancel"],
		noLink: true
	});

	app.quit();
})();
```

When you run the app above, a dialog appears. The caption and buttons are read, but not the message. The message used to be read with JAWS 2020, but not with 2021.

### Expected result
The message should be read.

### Actual result
The message isn't read.

#### Operating System and version
JAWS 2021.2102.34 ILM
Windows 10 20H2
Electron 12 (latest stable)

**Comment by pushkin-** [comment]
fixed in JAWS 2023
